misc/install.func

```diff
@@ -119,15 +143,15 @@ network_check() {
   ipv6_connected=false
   sleep 1
 # Check IPv4 connectivity
-  if ping -c 1 -W 1 1.1.1.1 &>/dev/null; then 
+  if ping -c 1 -W 1 1.1.1.1 &>/dev/null || ping -c 1 -W 1 8.8.8.8 &>/dev/null || ping -c 1 -W 1 9.9.9.9 &>/dev/null; then 
     msg_ok "IPv4 Internet Connected";
     ipv4_connected=true
   else
     msg_error "IPv4 Internet Not Connected";
   fi
 
 # Check IPv6 connectivity
```

Comment:
```suggestion
Check IPv6 connectivity to Cloudflare, Google, & Quad9 DNS servers.
```

Again, making it explicit

Reply:
I think its especially helpful because most people can't remember IPv6 addresses.